src/Microsoft.DotNet.XHarness.iOS.Shared/Hardware/SimulatorLoader.cs

```diff
@@ -83,7 +83,7 @@ public async Task LoadDevices(ILog log, bool includeLocked = false, bool forceRe
 
                 var xmlContent = File.ReadAllText(tmpfile);
 
-                log.WriteLine("Simulator listing returned:" + Environment.NewLine + xmlContent);
+                log.WriteLine($"Simulator listing finished ({Math.Ceiling(((double)Encoding.UTF8.GetByteCount(xmlContent)) / 1024)} kB)");
```

Comment:
we're not using `xmlContent` anymore except for getting this byte count so it seems expensive to do a `File.ReadAllText()` when we could just do a `new FileInfo(tmpfile).Length` instead.